app/view/Client/frmClient.cs

```diff
@@ -67,15 +75,23 @@ private void btnAddPatient_Click(object sender, EventArgs e)
             using (frmClientPatientModal newClientPatientForm = new frmClientPatientModal())
             {
                 newClientPatientForm.ShowDialog();
+                
             }
         }
 
         private void btnEditPatient_Click(object sender, EventArgs e)
         {
             using (frmClientPatientModal newClientPatientForm = new frmClientPatientModal())
-            {
-                newClientPatientForm.ShowDialog();
-            }
+
+                if (dgvPatient.RowCount > 0)
+                {
+                    int petId = Convert.ToInt32(dgvPatient.SelectedRows[0].Cells["Id"].Value);
+                    MessageBox.Show("Are you sure you want to UPDATE ?");
+                    frmClientPatientModal frm = new frmClientPatientModal();
+                    newClientPatientForm.ShowDialog();
+                    frm.ShowDialog();
```

Comment:
Duplicate instantiation of the same form class? This will open multiple clientPatient form after disposing the other one